packages/core/src/layout/HomepageTimer/HomepageTimer.tsx

```diff
@@ -39,18 +39,28 @@ function getTimes(configApi: ConfigApi) {
 
   for (const clock of clockConfigs) {
     if (clock.has('label') && clock.has('timezone')) {
+      let label = clock.getString('label');
+
       const options = {
         timeZone: clock.getString('timezone'),
         ...timeFormat,
       };
 
-      const time = d.toLocaleTimeString(lang, options);
-      const label = clock.getString('label');
+      try {
+        new Date().toLocaleString(lang, options);
```

Comment:
Thanks! Could you also add/update tests to reflect this?

Reply:
Oops - yup will get on that

Reply:
Cool, let me know when you've added a test.

Reply:
Added a test now - sorry for the delay, had to get my head around writing the test.